sdk/core/azure-core/azure/core/serialization.py

```diff
@@ -4,20 +4,103 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
+import base64
+import datetime
+from json import JSONEncoder
+
+from .utils._utils import _FixedOffset
```

Comment:
Do we really need to import a private type from a private submodule?

Reply:
I think we should rename _FixedOffset to FixedOffset. But not sure if it is worth...